source/includes/code-examples/InsertOne.cs

```diff
@@ -0,0 +1,95 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CsharpExamples.UsageExamples;
+
+public class InsertOne
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        WriteLine("Inserting a document...");
+        InsertOneRestaurant();
+
+        // find and print newly inserted document
+        var document = _restaurantsCollection.Find(Builders<Restaurant>.Filter
+            .Eq("name", "Mongo's Pizza")).First();
+
+        WriteLine("Document Inserted: " + document.ToBsonDocument());
+
+        Cleanup();
+
+    }
+
+    private static void InsertOneRestaurant()
+    {
+        // delete sample document if already exists
+        Cleanup();
+
+        // start-insert-one
+        Restaurant newRestaurant = new()
+        {
+            Name = "Mongo's Pizza",
+            RestaurantId = "12345",
+            Cuisine = "Pizza",
+            Address = new BsonDocument
+            {
+                {"street", "Pizza St"},
+                {"zipcode", "10003"},
+            },
+            Borough = "Manhattan",
+            Grades = new List<object>
+            {
+                new BsonDocument()
+            }
```

Comment:
issue: This doesn't seem to add much to the example, consider removing it. This suggestion applies to the async version as well.